templates/default.conf

```diff
@@ -74,7 +74,7 @@ server {
         try_files $uri @rewrite;
     }
 
-    location ~* \.(txt|log)$ {
+    location ~* \.(log)$ {
```

Comment:
i need this because in contexts like `http://drupal.loc/system/files/2023-11/test.txt` nginx will block the request, and txt files can be used as valid assets.